File: Practicals/07_11/Vowel.java
/*
 *******************************************************************************
 *  This program saprates the Vowels and Consonents from input string.
 *******************************************************************************
*/

import java.util.Scanner;

class DiffChar {
    public static void main(String[] args) {
        Scanner str = new Scanner(System.in);
        String string = str.nextLine();
        String vowel = "";
        String consonant = "";

        for(int i = 0; i < string.length() - 1;i++ ) {
            char ch = string.charAt(i);

            if(ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u')
                vowel = vowel + ch + " ";

            else if(ch == ' ')
                continue;

            else
                consonant = consonant + ch + " ";
        }

        System.out.println("Vowels     : " + vowel);
        System.out.println("Consonents : " + consonant);
    }
}

File: Practicals/07_11/Python/Vowel.py
def main():
    str = input("Enter your string : ")
    vowel = ""
    consonent = ""
    for ch in str:
        if ch in {'a','e','i','o','u','A','E','I','O','U'}:
            vowel = vowel + ch + " "
        elif ch == ' ':
            continue
        else:
            consonent = consonent + ch + " "

    print("Vowels     : ", vowel)
    print("Consonents : ", consonent)

if __name__ == '__main__':
    main()
